### ISSUE TITLE
How to compare with vue-strap?

### ISSUE THREAD

**Issue by ryanaponte** [kzima/vuestrap#1]
I love what I see, congrats! But what should a developer be taking into consideration when deciding between `vuestrap` and `vue-strap`? Thanks

**Comment by kzima** [comment]
@ryanaponte Thanks for your feedback. 

Vustrap is just a simple js wrapper for Bootstrap 4 and can be easily imported via webpack or browserfy in projects that use frontend frameworks like jQuery, Angular or Ember. But the idea behind Vuestrap is to create a foundation to component libraries built with Vue.js in particular, hence the name :)

Benefits of using Vuestrap over yuchte/vuestrap:
- It uses Bootstrap 4 source files and compile them on the fly in your project. That means smaller footprint if you only need few components in your project.
  _In yuchte/vue-strap you need to load a full boostrap css file, or have to have a separate build process in place (like grunt or gulp)._
- It allows your components to be truly themable, so if you set `$brand-primary` to 'red' in your application theme, all the components that use this variable will compile to 'red', neat! 
  _In yuchte/vue-strap you would have to override every single component. This is because it's components rely on external boostrap css file that is already compiled._
- Finally, most of components built with Vuestrap support IE9 by default and can be used for commercial projects that require a better browser support.
  _It is hard to tell if yuchte/vue-strap will support IE9 in the near future (I could not find in the roadmap)._

In summary:
Vuestrap and yuchte/vuestrap are based on similar concept, but if you need more flexibility in your development or want to [create](http://kzima.github.io/vuestrap-docs/#/) your own custom components, then probably you want to use Vuestrap.

**Comment by ryandeussing** [comment]
Thanks, very cool.
